README.md

```diff
@@ -63,7 +63,7 @@ Of course there is a lot of convenience integrated into it to make integration t
 The requirements to write integration tests with the integration testing framework are the folloing:
 
 * JDK11+
-* Apache Maven 3.1.0 or above. (I recommend to use Maven 3.8.1)
+* Apache Maven 3.1.0 or above. (I recommend to use Maven 3.8.2)
```

Comment:
Unrelated change; let me know if you'd like to see this reverted. (Or perhaps rephrased to "I recommend using the latest release".)